python/hail/expr/functions.py

```diff
@@ -640,6 +658,23 @@ def parse_locus(s, reference_genome=None):
     return construct_expr(ApplyMethod('Locus({})'.format(reference_genome.name), s._ast), TLocus(reference_genome),
                           s._indices, s._aggregations, s._joins, s._refs)
 
+@typecheck(pl=ArrayNumericExpression)
+def pl_dosage(pl):
```

Comment:
Does these two functions need to get added to an `__init__.py` file?

Reply:
I don't think so, since functions is a single-file module

Reply:
Right. I added these because the old doc examples for skat used them. I ported the examples to the new docstring, which uses both functions, and the tests pass.